src/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -170,13 +170,18 @@ private function getResourceCollectionConfig()
         $tree = new TreeBuilder();
         $subTree = $tree->root('resource_collection');
         $subTree->addDefaultsIfNotSet();
+        $outboxUrl = rtrim(URL_OUTBOX, '/');
+        // the outboxurl may be an absolute url - we want a relative url instead. Reason:
+        // 1. if the container is build using the console, then the URL_OUTBOX will have "console" as a domain
+        // 2. The URL is added via \TGlobalBase::GetStaticURL which will take care of adding the correct url.
+        $outboxRelativeUrl = parse_url($outboxUrl, PHP_URL_PATH);
```

Comment:
-> $outboxUrlPath 

(it's not really an url anymore)

Reply:
renamed to `$defaultCacheUrlPath`